tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java

```diff
@@ -55,8 +54,7 @@
       ElasticsearchConnector.class
     },
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ContextConfiguration(initializers = AddManagementPropertiesInitializer.class)
-@ActiveProfiles(AUTH_PROFILE)
+@ActiveProfiles({AUTH_PROFILE, "tasklist", "test"})
```

Comment:
[Question] Why we are using 3 active profiles over here?

Reply:
- `AUTH_PROFILE` is required by this test scenario 
- `tasklist` is to import the properties in `application-tasklist.properties` (in `dist` folder), the goal is to test similar configuration to the one that is used by the distribution
- `test` is to import the properties in `applicaiton-test.yml` (some specific properties that is used for tests like fake static files...)

Reply:
Thanks for clarifying this. 💯